packages/big-design/src/components/FileUploader/File.tsx

```diff
@@ -1,18 +1,22 @@
-import { DeleteIcon, DraftIcon } from '@bigcommerce/big-design-icons';
+import { DeleteIcon, DraftIcon, MoreHorizIcon } from '@bigcommerce/big-design-icons';
 import React, { HTMLAttributes, useMemo } from 'react';
 
 import { Button } from '../Button';
+import { Dropdown } from '../Dropdown';
+import { DropdownItem } from '../Dropdown/types';
 
 import { FileStyled, ImageStyled, TextEllipsed } from './styled';
 
 interface Props extends HTMLAttributes<HTMLDivElement> {
-  name: string;
+  actions?: DropdownItem[];
```

Comment:
Do you think we would need other actions than removing the file? If we don't, can we just handle this for the user instead?

Reply:
@vdm-kr, could you please explain, why we decided to use `InlineMessage` for form component, and do we actually need additional `actions` other than deletion?

Reply:
My comment on using `InlineMessage` still stands, @vdm-kr can we update the designs to go with our current pattern of displaying form error messages?

Reply:
Same for the error message since it could be used not only in form and files could be rendered separately we need to display errors in a more universal way. 
![Errors](https://github.com/bigcommerce/big-design/assets/30108921/2bb90c92-55ac-4de3-9d38-1de96ccd89bc)

Reply:
@vdm-kr the `InlineMessage` component was only meant to be used within `Modal`'s. I remember we worked with design to come up with that guideline:
![Screenshot 2024-03-05 at 09 36 25](https://github.com/bigcommerce/big-design/assets/10539418/17197740-358d-493a-b5b8-08452f5f3e61)

Reply:
<img width="1035" alt="Screenshot 2024-03-11 at 08 36 26" src="https://github.com/bigcommerce/big-design/assets/30108921/86eb6cca-4ccf-456c-8c88-56da8eaeae5e">
In the guidelines below we do allow to place it inside panels. I also discussed it one more time with David, Eugene, and Turner, they confirmed

Reply:
We probably need to update that description then if they can be used in both. Now that I think about it, we have used it in both places.

@vdm-kr The big issue I see here is that now we have two different ways of showing errors within form controls. Since a `FileUploader` is a type of form control, those will need to work within forms. `InlineMessage` components were _originally_ meant to be used at the top of modals as a more condensed way of showing a message. Now that they have been expanded to be used within pages, there are now two different ways of showing errors within forms. As a pattern library we are trying to enforce a consistency for both our merchants and our developers (including 3rd party devs). So as a developer, trying to enforce a consistent design pattern, what should we use for showing form validation issues now? `InlineMessage` or the errors underneath the form fields? I know there's probably use-cases for one or the other, but this will make a confusing pattern for consumers of this.

![Screenshot 2024-03-14 at 12 01 51](https://github.com/bigcommerce/big-design/assets/10539418/e99da0fa-2813-446f-a79d-bb9093d2b751)

Reply:
Let’s update description to “Used for displaying alerts within Panels and Modals. Is a condensed version of the Message component.”

I think it’s okay to have both since fields have different behaviour. File uploader doesn’t look and work in the same way as a text field. Also, this error states are embedded in the component so I don’t really think there will be any issues because developers won’t need to decide how to display errors.